pgcli/pgexecute.py

```diff
@@ -123,21 +123,19 @@ def run(self, sql):
                 return [(None, None, cur.statusmessage)]
 
     def tables(self):
```

Comment:
Let's rename this to def tables_columns(self) to reflect what it's actually doing.